docs/semgrep-ci/running-semgrep-ci-with-semgrep-app.md

```diff
@@ -328,11 +329,107 @@ SEMGREP_PR_ID="44"
 
 ### Receiving PR or MR comments
 
+To receive PR or MR comments in your repository, follow the steps to enable hyperlinks. Verify that comments are sent by adding rules to your Rule Board's **Comment** column that can match code to generate a finding.
+
 :::info
-* Semgrep Cloud Platform does not support PR comments within BitBucket repositories.
+Only rules that are in the **Comment** column of your [Rule board](https://semgrep.dev/orgs/-/board) create the PR comments.
```

Comment:
Rules in either of the Comment or Block column of the Rule Board can cause PR comments to fire. The difference is that findings from the Comment column will not cause CI to be blocked while a blocking finding would.